### PR TITLE
changes topo gen to move topo in z direction [in3Utils]

### PR DESCRIPTION
**Pull request by chhesselbach**

### CODE REVIEW

File: avaframe/in3Utils/generateTopo.py [hunk 1]
```diff
@@ -676,6 +676,11 @@ def generateTopo(cfg, avalancheDir):
     # If a drop shall be introduced
     if cfg['TOPO'].getboolean('drop'):
         z = addDrop(cfg, x, y, z)
+    
+    # moves topo in z direction 
+    if cfg['DEMDATA']['zEdit'] != '':
+    	z = z+cfg['DEMDATA'].getfloat('zEdit')
```
Comment (fso42) [inline avaframe/in3Utils/generateTopo.py:682]: Please make sure indentation is done via spaces and not tabs (lines 682 and 683) 

I recommend to change this in your editor (so spaces are used everytime you tab..)

File: avaframe/in3Utils/generateTopoCfg.ini [hunk 2]
```diff
@@ -131,3 +131,9 @@ demName = DEM
 # no data value
 nodata_value = -9999
 #---------------------------------------------------------
+
+# moves topo in z direction [m]
```
Comment (fso42) [inline avaframe/in3Utils/generateTopoCfg.ini:135]: Maybe add something like: e.g +100 or -100

File: avaframe/in3Utils/generateTopo.py [hunk 3]
```diff
@@ -676,6 +676,11 @@ def generateTopo(cfg, avalancheDir):
...
+    	log.info('Changed topo elevation by zEdit')
```
Comment (fso42) [inline avaframe/in3Utils/generateTopo.py:683]: If you want you could give the value for zEdit as well, might be helpful...

But that's only if you want to...

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/in3Utils/generateTopo.py (hunk 3 above)
Comment (awirb) [reply avaframe/in3Utils/generateTopo.py:683]: I think that is a good idea- so you could change it to: log.info('Changed topo elevation by %.2f' % cfg['DEMDATA'].getfloat('zEdit'))

**Review by awirb — APPROVED** [review]

**Comment by awirb** [comment]
Consider adding a pytest

**Review by fso42 — APPROVED** [review]

**Pull request merged**